<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Choose_course</class>
 <widget class="QWidget" name="Choose_course">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>280</width>
     <height>257</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="English">
            <property name="text">
             <string>英语口语</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Physics">
            <property name="text">
             <string>大学物理</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Programming">
            <property name="text">
             <string>程序设计</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Computer_theory">
            <property name="text">
             <string>计算机原理</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="OOP_programming">
            <property name="text">
             <string>OOP程序设计</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Microcomputer">
            <property name="text">
             <string>微机原理</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="Mathematics">
            <property name="text">
             <string>高等数学</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Politics">
            <property name="text">
             <string>政治经济学</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Data_structure">
            <property name="text">
             <string>数据结构</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Assembly_language">
            <property name="text">
             <string>汇编语言</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Complie_fud">
            <property name="text">
             <string>编译原理</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Comp_networks">
            <property name="text">
             <string>计算机网络</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="Confirm_course">
          <property name="text">
           <string>OK</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="back_2">
          <property name="text">
           <string>back</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
